### DISCUSSION TITLE
Overriding action in subclass gives "******@************** is annotated with 'override', but no such annotated member was found on prototype." error

### DISCUSSION THREAD

**Discussion by camondrey** [mobxjs/mobx#3003] [Q&A] [answered]
Following the guide at https://mobx.js.org/subclassing.html, I have created a parent class and a subclass that inherits from it. My goal is to have a parent class that contains a number of actions, observables, and so forth, and then give the subclass the option to override a few of them and also add some new ones.

More concretely, suppose that this is my parent class:

```
class Store {
  tooltip = null

  constructor() {
    makeObservable(this, {
      setTooltip: action.bound,
      removeTooltip: action.bound
    })
  }

  setTooltip(data) {
    this.tooltip = data
  }

  removeTooltip() {
    this.tooltip = null
  }
}
```

My subclass is the following:

```
class SubStore extends Store {
  constructor() {
    super()
    makeObservable(this, {
      setTooltip: override
    })
  }

  setTooltip(data) {
    this.setTooltip({ ...data, newProperty: true })
  }
}
```

I'm guessing there is something I've overlooked as I get an error stating that `******@************** is annotated with 'override', but no such annotated member was found on prototype.` and I can't figure out what the issue is. I'm stumped as I thought this was the correct way to set it up.

I'd appreciate some help with this. Thank you.

**Comment by urugator** [comment, marked as answer]
Can't reproduce https://codesandbox.io/s/nameless-sky-1gt9c?file=/index.js

**Reply by camondrey** [reply]
I see - In that case, does that mean that I set it up right but for some reason it doesn't work in my local environment?

**Reply by urugator** [reply]
If this is 1:1 copy of your actual code, then it should be the case...

**Reply by camondrey** [reply]
The code above is a simplified version of my real code - I just tested the example code here and in fact it also works fine for me like in your sandbox, which means that the issue must be stemming from something else (in my actual code)! I will have to go hunt it down, but in any case thank you for the help and for confirming that I'm setting up my subclasses the right way. 👍

**Reply by felpin** [reply]
@camondrey do you remember how to fix it? I am going through something similar
